bloomfilter/src/Data/BloomFilter/BitVec64.hs

```diff
@@ -83,6 +98,13 @@ thaw (BV64 bv) = MBV64 <$> P.thaw bv
 roundUpTo64 :: Word64 -> Word64
 roundUpTo64 i = unsafeShiftR (i + 63) 6
 
+remWord64 :: Word64 -> Word64 -> Word64
+#if MIN_VERSION_base(4,17,0)
+remWord64 (W64# x#) (W64# y#) = W64# (x# `remWord64#` y#)
+#else
+remWord64 (W64# x#) (W64# y#) = W64# (x# `remWord#` y#)
+#endif
+
```

Comment:
They omit the check for 0.
```
instance Integral Word64 where
    ...
    rem     (W64# x#) y@(W64# y#)
        | y /= 0                    = W64# (x# `remWord64#` y#)
        | otherwise                 = divZeroError
```

Reply:
And we have this in the smart constructor:

```haskell
new hash numBits = MBloom hash numBits' `liftM` V.new numBits'
  where numBits' | numBits == 0                = 1
```

So we should be safe here. Maybe still a good idea to have a check, outside of the hot loop of course.

Reply:
Makes sense, thanks! I think it's a reasonable to assume bloom filters are non-empty